ql/src/test/results/clientpositive/llap/check_constraint.q.out

```diff
@@ -2415,14 +2415,14 @@ STAGE PLANS:
                 outputColumnNames: _col0, _col1, _col2, _col3, _col4, _col5
                 Statistics: Num rows: 1 Data size: 409 Basic stats: COMPLETE Column stats: NONE
                 Select Operator
-                  expressions: _col1 (type: string), _col0 (type: int), _col4 (type: string), _col5 (type: struct<writeid:bigint,bucketid:int,rowid:bigint>), _col2 (type: string), _col3 (type: int)
+                  expressions: _col0 (type: int), _col4 (type: string), _col3 (type: int), _col1 (type: string), _col5 (type: struct<writeid:bigint,bucketid:int,rowid:bigint>), _col2 (type: string)
```

Comment:
Are these q.out changes expected? What is the root cause?

Reply:
the join operand order seem to have changed